pages/actions.tsx

```diff
@@ -56,6 +57,14 @@ export function ActionsPage() {
       .order("id", { ascending: true })
       .eq("org_id", profile?.org_id!);
 
+    const modelRes = await supabase
+      .from("organizations")
+      .select("model")
+      .eq("id", profile?.org_id!);
```

Comment:
This should already be in `profile` so basically all the logic you've added here can be removed :smile: